### DISCUSSION TITLE
Anomaly

### DISCUSSION THREAD

**Discussion by aliyafarhana** [Lost-MSth/Arcaea-server#121] [Q&A]
Hi, I have a question, how I can create my own anomaly by using tempest anomaly, finale anomaly, ether strike anomaly, arcahv anomaly and magnolia anomaly? Don't misunderstand, what I actually mean is, how I can use one of the anomaly for custom song? Because I really like anomaly in Arcaea, so I really really need it.

**Comment by innocentvoice** [comment]
Modify the game binary by yourself~

> everything is open source if you can reverse engineer -Low Level Learning

**Reply by innocentvoice** [reply]
Or better, Use Arccreate.
